.dockerignore

```diff
@@ -0,0 +1,24 @@
+**/.classpath
```

Comment:
I would prefer a dockerignore files that ignores relevant project files (e.g. wav files, README.md).

In this case, lots of the lines are seemingly related to your own setup (vscode, dbmdl, azds). I don't know what these words are, presumably Visual Studio + Azure-related?

Unfortunately there is no global .dockerignore (the way you can set up a global .gitignore) so I don't have a better suggestion for you: https://github.com/moby/moby/issues/12843

Reply:
Alternatively, you can negate .dockerignore entries (https://docs.docker.com/engine/reference/builder/#dockerignore-file) - so maybe a better approach is to ignore everything except some important project files.

Something like:
```
*
!src
```

Reply:
Yeah that was just the boilerplate .dockerignore that the Docker VSCode extension generates, I didn't modify it. Let me look and see what can be omitted.

Reply:
I just updated it, I left some things like build files, etc. because I am using the image for development purposes (rather than prod) so having those helps me.

Reply:
FYI: docker pull esimkowitz/pitchdetection